example/src/main/java/org/nervos/ckb/transaction/CollectResult.java

```diff
@@ -0,0 +1,14 @@
+package org.nervos.ckb.transaction;
+
+import java.util.List;
+
+/** Copyright © 2019 Nervos Foundation. All rights reserved. */
+public class CollectResult {
+  public List<CellsWithAddress> cellsWithAddresses;
+  public String changeCapacity;
```

Comment:
should be `BigInteger` ?

Reply:
This filed data structure is hex string and `BigInteger` is also ok. I think they are all ok.

Reply:
To be same as other class, I will update it.